.github/workflows/go.yml

```diff
@@ -4,10 +4,18 @@ on:
     branches:
     - master
     - release-*
+    paths-ignore:
```

Comment:
@lzhecheng did you confirm that when using `paths-ignore`, Github reports "success" as the status for the PR? I cannot find information online about this.

Reply:
I tested it on my repo. I changed kind.yaml and all kind tests are gone.
https://github.com/lzhecheng/antrea/pull/125

Reply:
I think my point is that our Github is currently configured to only allow merging a PR if all Go checks have passed (plus some others). If we use `paths-ignore`, the Go checks will not run and the status will not be reported as "success", which means merging will be blocked for the PR. I think you can test it in your fork by updating the branch protection settings.

Reply:
Thanks. I didn't know the protection settings. I will keep looking into this issue.